sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -578,6 +669,15 @@ class HistGradientBoostingRegressor(BaseHistGradientBoosting, RegressorMixin):
     verbose: int, optional (default=0)
         The verbosity level. If not zero, print some information about the
         fitting process.
+    warm_start : bool, optional (default=False)
+        When set to ``True``, reuse the solution of the previous call to fit
+        and add more estimators to the ensemble, otherwise, just erase the
```

Comment:
I would remove the "otherwise" part

Also I think that this would be enough (no need to go case-by-case) "For results to be valid, the estimator should be re-trained on the same data only."